Source/VCGeneration/Split.cs

```diff
@@ -15,6 +15,14 @@ namespace VC
   using Bpl = Microsoft.Boogie;
   using System.Threading.Tasks;
 
+  public record VCResult
```

Comment:
This change overall LGTM, but it made me realize that we're not yet recording the resource count for splits, only for methods. :( Can we fix that in the next PR?

Reply:
That just occurred to me independently, too. And, yes, I'd very much like to add that ASAP.